solutions/fully-configurable/main.tf

```diff
@@ -378,10 +379,10 @@ locals {
 
```

Comment:
`create_sm_auth_policy` -> `create_secrets_manager_auth_policy`


do we need depends_on here? redis doesnt have it 
```
resource "ibm_iam_authorization_policy" "secrets_manager_key_manager" {
  count                       = local.create_sm_auth_policy
  depends_on                  = [module.elasticsearch] 
   ```